nemo/launch.sh

```diff
@@ -55,19 +58,20 @@ export ADDITIONAL_ARGS="++model.micro_batch_size=$MICRO_BATCH ++trainer.max_step
 # == construct job launch command == 
 
 # create base job launch command 
-export LAUNCH_CMD="git clone https://github.com/hosseinsarshar/dist-training-vertex.git &&"
+export LAUNCH_CMD="git clone -b sync-copy-mpi https://github.com/hosseinsarshar/dist-training-vertex.git &&"
```

Comment:
just need to remove the branch reference

Reply:
fixed in this commit e0b885d